tests/Properties.hs

```diff
@@ -82,36 +91,54 @@ prop_lines_lazy2 =
 
 prop_strip x = C.strip x == (C.dropSpace . C.reverse . C.dropSpace . C.reverse) x
 
--- Ensure that readInt and readInteger over lazy ByteStrings are not
+class (Bounded a, Integral a, Show a) => RdInt a where
+    rdIntC :: C.ByteString -> Maybe (a, C.ByteString)
+    rdIntD :: D.ByteString -> Maybe (a, D.ByteString)
```

Comment:
`C` and `D` are terrible abbreviations for `Data.ByteString.Char8` and `Data.ByteString.Lazy.Char8`. We should change this in a follow-up PR.

Reply:
I have opened https://github.com/haskell/bytestring/issues/439 about this.